src/settings.cpp

```diff
@@ -426,6 +426,32 @@ s16 Settings::getS16(const std::string &name) const
 }
 
 
+v3s16 Settings::getV3S16(const std::string &name) const
+{
+	v3s16 value;
+	Strfnd f(get(name));
+	f.next("(");
+	value.X = stoi(f.next(","), -32768, 32767);
+	value.Y = stoi(f.next(","), -32768, 32767);
+	value.Z = stoi(f.next(")"), -32768, 32767);
+	return value;
+}
+
+
+v3s16 Settings::getMapGenerationLimit() const
```

Comment:
I personally don't feel as if Settings is the appropriate place for this function from an organizational standpoint.

Could it be made a static method of Map instead, perhaps??